### ISSUE TITLE
Leaked 90 degrees rotation augmentations 

### ISSUE THREAD

**Issue by jonilaserson** [rosinality/stylegan2-pytorch#265]
I'm training on a smaller version of 256x256 FFHQ with only 3000 faces.
`python -m torch.distributed.launch --nproc_per_node=2 train.py --batch 8 --augment --wandb /workspace/data/ffhq3k-lmdb/ `

The emitted samples have some faces that are rotated by 90 degrees. I thought ADA was supposed to handle that? What's the best practice to handle this?
![image](https://user-images.githubusercontent.com/5720840/136929707-4b0e25c8-ccbc-4543-9562-75f034437644.png)

This is the augment metric:
![image](https://user-images.githubusercontent.com/5720840/136929922-0fca2cea-1691-448f-b6a4-52ad4d1d1478.png)
